middleware/throttle.go

```diff
@@ -34,21 +54,46 @@ type token struct{}
 
 // throttler limits number of currently processed requests at a time.
 type throttle struct {
```

Comment:
that convention is for interfaces.. not for middlewares. I don't buy that argument :)